hera_cal/lstbin.py

```diff
@@ -62,6 +63,12 @@ def lst_bin(data_list, lst_list, flags_list=None, dlst=None, begin_lst=None, lst
     freq_array : type=ndarray, 1D array of unique data frequencies channels in Hz. Needed for rephase.
     lat : type=float, latitude of array in degrees North. Needed for rephase.
     verbose : type=bool, if True report feedback to stdout
+    nsamp_list : list of nsamples arrays
+    bl_list : optional list of antenna pairs that includes baselines that may not be in the data for the chunk of lsts
```

Comment:
Does it only take antenna pairs or does it take bl tuples like `(0, 1, 'ee')`?

Reply:
antenna pairs. I've clarified this.